### DISCUSSION TITLE
Line Tools In Action (Video)

### DISCUSSION THREAD

**Discussion by difurious** [difurious/lightweight-charts-line-tools#3] [Announcements]
https://github.com/difurious/lightweight-charts-line-tools/assets/61764595/8a0df975-9d20-4971-be8a-9ca9531e20ae

**Comment by sasax7** [comment]
Hi is there a way to import the chart you show in the video in ones own project?

**Reply by difurious** [reply]
Im not sure I follow your question. By using the line tools build you can use those same tools on the chart.  The UI buttons are part of a react frontend, but they are just bottoms that run the functions that create the line tools.  You would need to code your own front end for however you want the buttons to look - work.  Mine is not public.
